### DISCUSSION TITLE
Chrome, TBS stopped recording sessions at exit, window close, regular save.  Only retains one (1) manually saved session

### DISCUSSION THREAD

**Discussion by mybasementcloud** [sienori/Tab-Session-Manager#1349] [Q&A]
I just noticed that TBS has stopped recording sessions at exit, window close, regular save on my primary Chrome profile--the second profile is working as expected.  Also, that primary Chrome profile session set only retains one (1) manually saved session, and removes any previous manually saved sessions.  TBS is updating as expected, but not saving sessions as expected or configured.

Is there a process to generate debug data and provide for review.

**Comment by LYCAON68** [comment]
Hello,

With latest updated Chrome on Win11, i have the same problem with no local sessions recording (automatically or manually) on a well configurated DownLoad Location with it's Sub Dirs : "TabSessionManager_Backup", they are all empty ! ...

Is it possible to see somewhere the saved sessions on the Google Drive, i can't see any ?

Best Regard's

**Comment by awprawpr** [comment]
Does the lack of replies imply the developer has deserted the app?

**Comment by mybasementcloud** [comment]
I don't believe so, because FOSS is much a product of "developer has time" development operation.
What this project is missing is a clear use of the Issues option on GitHub to collect issues, and also provide methods for initial troubleshooting when things don't work as expected.

**Comment by mybasementcloud** [comment]
I've checked my other systems (different computers) and there TSM is working as expected, taking the defined session backups.
It's only this one Chrome instance (I have two on this system), and that instance is also apparently not properly synching.

What we need is some guidance on how to clean up the TSM add-on installation and data in a Chrome instance, so that the add-on can be added from scratch to see if that helps--AFTER taking a session copy of all Windows and Tabs and Exporting that json for import.

**Reply by mybasementcloud** [reply]
I stand corrected, Issues are in use, and I should have started this thread there, but for some reason didn't notice it before.
My bad.

**Comment by mybasementcloud** [comment]
Opened Issue here:
https://github.com/sienori/Tab-Session-Manager/issues/1358

**Comment by mybasementcloud** [comment]
Closing discussion to focus on Issue
